main.go

```diff
@@ -156,17 +157,17 @@ func collectStatsForWithError(proc *process.Process, withError bool) (*MozProces
 	/* Attempting to collect the CPU percentage as well to better query for the
 	instance usage.
 	*/
-	cpu_percent, err := proc.CPUPercent()
+	cpuPercent, err := proc.CPUPercent()
```

Comment:
Oups, sorry for not being consistent here, my bad.

Reply:
New language, new code standards, it's all fine